common/beerocks/bwl/nl80211/ap_wlan_hal_nl80211.cpp

```diff
@@ -451,122 +451,49 @@ bool ap_wlan_hal_nl80211::read_supported_channels()
     m_radio_info.supported_channels.clear();
     // Resize the supported channels vector
     m_radio_info.supported_channels.resize(MAX_SUPPORTED_20M_CHANNELS);
```

Comment:
same comment here regarding the _20M_

Reply:
I've explained in the first comment, why we need to use `MAX_SUPPORTED_20M_CHANNELS`